signer/src/transaction_signer.rs

```diff
@@ -488,10 +498,17 @@ where
                     self.signer_private_key,
                 )?;
                 self.wsts_state_machines.insert(msg.txid, state_machine);
+                // Let's give the others some slack
+                tracing::debug!(
+                    "Sleeping a bit to give the other peers some slack to get DkgBegin"
+                );
+                tokio::time::sleep(Duration::from_secs(10)).await;
```

Comment:
This may make some of our tests fail, maybe make this configurable or something?

Reply:
[147f3f3](https://github.com/stacks-network/sbtc/pull/920/commits/147f3f3c3816765267fcb735c1aeda1c5769228e)

Updated #947 as well in-case this ends up lingering around, but added a comment that it may become deprecated with upcoming dkg changes.

Reply:
🙏🏽